source/Engine.h

```diff
@@ -198,16 +211,22 @@ class Engine {
 	std::condition_variable condition;
 	std::mutex swapMutex;
 
-	bool calcTickTock = false;
-	bool drawTickTock = false;
+	// ES uses a technique called double buffering to calculate the next frame and render the current one simultaneously.
+	// To facilitate this, it uses two buffers for each list of things to draw - one for the next frame's calculations and
+	// one for the rendering the current frame. A little synchronisation is required to prevent mutable references to the
+	// currently rendering buffer.
```

Comment:
```suggestion
	// ES uses a technique called double buffering to calculate the next frame and render the current one simultaneously.
	// To facilitate this, it uses two buffers for each list of things to draw - one for the next frame's calculations and
	// one for rendering the current frame. A little synchronization is required to prevent mutable references to the
	// currently rendering buffer.
```